include/LLVMSPIRVOpts.h

```diff
@@ -238,6 +241,9 @@ class TranslatorOpts {
   }
   BuiltinFormat getBuiltinFormat() const noexcept { return SPIRVBuiltinFormat; }
 
+  void useLLVMTarget(bool Flag) noexcept { UseLLVMTarget = Flag; }
+  bool getUseLLVMSPIRVBackendTarget() const noexcept { return UseLLVMTarget; }
```

Comment:
May be rename UseLLVMTarget to be UseLLVMSPIRVBackendTarget everywhere (to be consistent)?

Reply:
Let's please rename the function only, please see https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2716#discussion_r1756820076 for the rationale

Reply:
```suggestion
  bool getUseLLVMTarget() const noexcept { return UseLLVMTarget; }
```

Reply:
thank you